README.md

```diff
@@ -1,46 +1,168 @@
 
-### Download and run the code
+## Installation
+
+### Prerequisites
+
+Make sure you have the following installed: 
+
+- [Node.js](https://nodejs.org) >= 7.0.0
+- [TypeScript](https://www.typescriptlang.org/) >= 2.0.0 `npm i -g typescript`
+- [TypeScript Node](https://github.com/TypeStrong/ts-node) >= 3.0.0 `npm i -g ts-node`
+
+### Install loopback-next
 
-**Install loopback-next**
 ```
-$ git clone git@github.com:strongloop/loopback-next
-$ cd loopback-next
-$ npm i
-$ npm i -g lerna
-$ lerna bootstrap
-$ npm link
+$ git clone git@github.com:strongloop/loopback-next
+$ cd loopback-next
+$ ./bin/build
 ```
 
-**Install loopback-next-example**
+### Install loopback-next-example dependencies
+
 ```
-$ cd ..
-$ git clone git@github.com:strongloop/loopback-next-example
-$ cd loopback-next-example
-$ npm i
-$ npm link loopback-next
-$ bin/start.sh
+$ cd ..
+$ git clone git@github.com:strongloop/loopback-next-example
+$ cd loopback-next-example
+$ ./bin/build
 ```
 
-### Make a request to get the account summary screen data, and get account.
+## Usage
+
+### Start all microservices
 
 ```
-$ bin/get-account-summary.sh
-$ bin/get-account.sh
+$ ./bin/start
 ```
 
+#### Sample Output
+
+```
+Stopping microservices related to the example...
+Starting services...
+Started account service, Pid: XXXXX
+Started customer service, PID: XXXXX
+Started transaction service, PID: XXXXX
+Started facade service, PID: XXXXX
+Application Info: { uptime: 14 }
+Application Info: { uptime: 15 }
+Application Info: { uptime: 13 }
+Application Info: { uptime: 11 }
+All microservices started successfully.
+To test the application, run the get account summary script:
+./bin/get-account-summary
+./bin/get-account
+```
+
+### Perform a HTTP GET request to retrieve account summary data
+
+```
+$ ./bin/get-account-summary
+```
+
+#### Sample Output
+
+```
+{
+  "account": {
+    "customerNumber": "000343223",
+    "balance": 85.84,
+    "branch": "Foster City",
+    "type": "Checking",
+    "avgBalance": 398.93,
+    "minimumBalance": 10,
+    "id": "CHK52321122"
+  },
+  "customer": [
+    {
+      "firstName": "Ron",
+      "lastName": "Simpson",
+      "ssn": "141-XX-X800",
+      "customerSince": "2017-03-14T23:05:18.779Z",
+      "street": "742 Evergreen Terrace",
+      "state": "OR",
+      "city": "Springfield",
+      "zip": "95555",
+      "lastUpdated": "2017-03-14T23:05:18.599Z",
+      "id": "000343223"
+    }
+  ],
+  "transaction": [
+    {
+      "TransactionId": "DEBIT0001",
+      "dateTime": "2017-03-11T00:27:52.422Z",
+      "accountNo": "CHK52321122",
+      "amount": 20,
+      "transactionType": "debit"
+    }
+```
+> The data above is the account summary for the Account with ID `CCHK52321122`.
+
+### Make a request to get the account
```

Comment:
"Perform a HTTP GET request to retrieve account data" for consistency with previous command